src/elf/mod.rs

```diff
@@ -53,6 +53,8 @@ pub mod dynamic;
 #[macro_use]
 pub mod reloc;
 pub mod note;
+#[cfg(all(any(feature = "elf32", feature = "elf64"), feature = "alloc"))]
```

Comment:
it's a little sad this only needs the `alloc` feature because (iiuc) it creates a `Strtab`, which I believe until only recently did not require the `alloc` feature either (maybe it still did, I can't remember).

Anyway, something we could possibly investigate fixing in the future, but it's not related to this PR.